src/test/java/org/kitteh/irc/client/library/implementation/KICLPropertiesTest.java

```diff
@@ -0,0 +1,12 @@
+package org.kitteh.irc.client.library.implementation;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class KICLPropertiesTest {
+
+    @Test
+    public void testVersion() throws Exception {
```

Comment:
This test confirms the kicl.properties in the test dir works, but what if the one in the main resources dir is modified differently? This test doesn't actually confirm it worked.